WordPress/Classes/ViewRelated/Reader/ReaderCardsStreamViewController.swift

```diff
@@ -10,6 +10,19 @@ class ReaderCardsStreamViewController: ReaderStreamViewController {
     /// Refresh counter used to for random posts on pull to refresh
     private var refreshCount = 0
 
+    /// Sorting option used for fetching cards
+    private var sortingOption: ReaderSortingOption = .popularity {
```

Comment:
if `sortingButton` has an accessible property `sortingOption`, why do we need another one here that just mirrors that one?

Reply:
Good point, I did not clean that after I added the button, tnx!